crates/runtime/src/lib.rs

```diff
@@ -348,7 +354,7 @@ impl_runtime_apis! {
                 tx_index as u32 == index
```

Comment:
A little bit not sure about this covertion from `u64` -> `u32`, but it was `usize` -> `u32`

Reply:
```Some((tx_index as u64, transaction))``` it's u64 because of this cast from usize.
I think you can cast index as u32 from the get go. There will never be more than u32::MAX transaction in a block.
So having index being a u32 everywhere sound ok to me. 

You can also use `try_from().expect()` if you don't want the error to be silenced and the number to saturate